GoogleForCreators/web-stories-wp
# Use SVG sprites for icons

## Summary
This PR adds support for using an svg sprite to render icons. The `svg-sprite-loader` webpack loader is used to generate the sprites.


## User-facing changes

None.

## Testing Instructions

This PR can be tested by following these steps:

1. Inspect any svg icon in the plugin.
2. You should see markup like this:
```html
<svg viewBox="0 0 24 24" width="22px">
  <use xlink:href="#homeWithHeart"></use>
</svg>
```


## Reviews

### Does this PR have a security-related impact?
No.

### Does this PR change what data or activity we track or use?
No.

### Does this PR have a legal-related impact?
No.

## Checklist




---

Fixes #13411
